src/url-loader/multi-url-resolver.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {UrlResolver} from './url-resolver';
+
+/**
+ * Resolves a URL using multiple resolvers.
+ */
+export class MultiUrlResolver implements UrlResolver {
+  constructor(private resolvers: Array<UrlResolver>) {
```

Comment:
let's name all private fields with a leading underscore, so there's some indication in compiled code.